maskhit/train.py

```diff
@@ -1,17 +1,29 @@
+"""
+Usage: python train.py --study=ibd --mil1=vit_h8l12 --mil2=ap --num-patches=0 
+--meta-svs=../../SlidePreprocessing/for_vit/meta/IBD_PROJECT/svs_meta.pickle 
+--meta-all=../../SlidePreprocessing/for_vit/meta/ibd_project_meta.pickle 
+--magnification=5 --lr-attn=1e-5 --lr-pred=1e-3 --wd=0.01 --outcome='Dx (U=UC, C=Cr, I=Ind)' 
+--outcome-type=classification --sample-patient --dropout=0.2 -b=4
+"""
+
 import os
 import time
+import ast
 import pandas as pd
 import sys
 import glob
 import socket
 from maskhit.trainer.fitter import HybridFitter
 from maskhit.trainer.losses import FlexLoss
 from options.train_options import TrainOptions
+from sklearn.model_selection import train_test_split
 
+print("\n")

```

Comment:
I see what you wanna do here but it could be more descriptive, by either generating a separator or add some message to explain the stage of this program.